src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  margin: 0;
+  display: flex;
+  width: 198px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  background-color: $color-white;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    margin: 32px 0 0;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 40px 0 0;
+    color: $color-main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    @include content-padding-inline;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+    align-self: flex-start;
+    color: $color-secondary;
+    font-size: 10px;
+
+    @include content-padding-inline;
+  }
+
+  &__reviews {
+    margin: 16px 0 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 166px;
+    height: 16px;
+
+    &-count {
+      margin: 0;
+      color: $color-main-accent;
+      font-size: 10px;
+    }
+  }
+
+  &__price {
+    margin: 24px 0 0;
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      margin: 0;
+      color: $color-secondary;
+    }
+
+    &-amount {
+      margin: 0;
+      color: $color-main-accent;
+      text-align: right;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    margin: 16px;
+    box-sizing: border-box;
+    text-decoration: none;
+    text-transform: uppercase;
+    display: block;
+    padding: 11px 68.5px;
+    width: 166px;
```

Comment:
```suggestion
    width: 100%;
```

Reply:
In my opinion, there's no reason to paste 100% here. Button is a block element, so it would only crash the inline padding, which I can't regularly change by `@include content-padding-inline;` If I messed something, please let me know and describe what I should do then.
![Screenshot 2023-12-17 192056](https://github.com/mate-academy/layout_product-cards/assets/89228879/9965b5df-af8c-4b5a-8880-45992e2a9297)
.